Pipfile

```diff
@@ -18,4 +18,4 @@ whoosh = "*"
 django-webpack-loader = "*"
 
 [requires]
-python_version = "3"
+python_version = "3.8"
```

Comment:
```suggestion
python_version = ">=python 3.8"
```

Reply:
You do not want to restrict the version to only 3.8. You just want to make sure the version is at least 3.8